samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -113,6 +124,8 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       formFactor={isMobileSession ? 'mobile' : 'desktop'}
       /* @conditional-compile-remove(rooms) */
       role={role}
+      /* @conditional-compile-remove(call-readiness) */
+      options={{ onPermissionsTroubleshootingClick, onNetworkingTroubleShootingClick }}
```

Comment:
I'd remove these or make them `alert`s instead of console logs, otherwise they seem like buttons that don't work

Reply:
fixed